chore: add customer info to usage event exports (#233)
langchain-ai/docs: src/langsmith/script-running-pg-support-queries.mdx

Exporting usage data requires running Helm chart version 0.11.4 or later.
</Note>

### Get customer information

> Before running the export scripts, you'll need to retrieve your customer information from the LangSmith API. This information is required as input for the export scripts.

Reviewer: ```suggestion
You need to retrieve your customer information from the LangSmith API before running the export scripts. This information is required as input for the export scripts.
```

```bash
curl https://<langsmith_url>/api/v1/info
# if configured with a subdomain / path prefix:
curl http://<langsmith_url/prefix/api/v1/info
```

This will return a JSON response containing your customer information:

```json
{
  "version": "0.11.4",
  "license_expiration_time": "2026-08-18T19:14:34Z",
  "customer_info": {
    "customer_id": "<id>",
    "customer_name": "<name>"
  }
}
```

Extract the `customer_id` and `customer_name` from this response to use as input for the export scripts.

### Process the API response with jq

You can use [jq](https://jqlang.org/download) to parse the JSON response and set bash variables for easy use in your scripts:

```bash
# Get the API response and extract customer information
export LANGSMITH_URL="<your_langsmith_url>"
response=$(curl -s $LANGSMITH_URL/api/v1/info)

# Extract customer_id and customer_name using jq
export CUSTOMER_ID=$(echo "$response" | jq -r '.customer_info.customer_id')
export CUSTOMER_NAME=$(echo "$response" | jq -r '.customer_info.customer_name')

# Verify the variables are set
echo "Customer ID: $CUSTOMER_ID"
echo "Customer Name: $CUSTOMER_NAME"
```

> These environment variables can then be used in your export scripts or other commands.

Reviewer 2: ```suggestion
You can then use these environment variables in your export scripts or other commands.
```
